docs/developers/guides/run-linea/run-the-stack/local.mdx

```diff
@@ -0,0 +1,116 @@
+---
+description: Run the Linea stack locally.
+image: /img/socialCards/run-the-linea-stack-locally.jpg
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Run locally
+
+## Prerequisites
+
+### Hardware requirements
+
+TODO
```

Comment:
@naveen-consensys and @bradbown, do either of you perhaps have info regarding the hardware requirements?

Reply:
I don't have anything concrete, but I supply docker with 10 cpu cores and 8gb of memory and 1 gb of swap, and it works but if I had more available resources I'd give it more, but I only have a 16gb laptop.